lib/pi_piper/sysfs/sysfs.rb

```diff
@@ -0,0 +1,73 @@
+module PiPiper
+  class Sysfs < Driver
+
+    def initialize
+      @exported_pins = Set.new
+    end
+
+    def close
+      unexport_all
+      @exported_pins.empty? 
+    end
+    
+# Support GPIO pins
+    GPIO_HIGH = 1
+    GPIO_LOW  = 0
```

Comment:
Hm, that should be fine regarding methods. But for constants, those should still be at the top. You can certainly space them out with comments though. Like this:

```ruby
class Sysfs < Driver
  # Pin/GPIO constants
  GPIO_HIGH = 1
  GPIO_LOW = 0

  # Constants for something else
  BLAH = true

  def initialize
  end
...
end
```